dojo/unittests/tools/test_intsights_parser.py

```diff
@@ -0,0 +1,53 @@
+from django.test import TestCase
+from dojo.tools.intsights.parser import IntSightsParser
+from dojo.models import Engagement, Product, Test
+
+
+class TestIntSightsParser(TestCase):
+    def get_test(self):
+        test = Test()
+        test.engagement = Engagement()
+        test.engagement.product = Product()
+        return test
+
+    def test_intsights_parser_without_file_has_no_findings(self):
+        parser = IntSightsParser()
+        findings = parser.get_findings(None, self.get_test())
+        self.assertEqual(0, len(findings))
```

Comment:
```suggestion
```
This use case is prohibited now. the upper layers will not call this method without a file.

Reply:
@damiencarol This is also part of the boilerplate cookie cutter code. Should it be removed?

Reply:
yep. I see that I really need to update this boilerplate code :D